codegenerator/cli/templates/dynamic/codegen/src/IO.res.hbs

```diff
@@ -137,9 +137,10 @@ let executeDbFunctionsEntity = (
 }
 
 let executeBatch = async (sql, ~inMemoryStore: InMemoryStore.t) => {
-  let entityDbExecutionComposer = Config.getGenerated()->Config.shouldRollbackOnReorg
-    ? executeSetEntityWithHistory
-    : executeDbFunctionsEntity
+  let entityDbExecutionComposer =
+    RegisterHandlers.getConfig()->Config.shouldRollbackOnReorg
```

Comment:
Config.getGenerated is removed, it now needs to be accessed from RegisterHandlers.getConfig() to ensure handlers are all registered (with event options) before instantiating the workers